unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTAsset/NFTAssetLoadHelper.cs

```diff
@@ -22,7 +21,7 @@ public class NFTAssetRetriever : INFTAssetRetriever
         private const string CONTENT_LENGTH = "Content-Length";
         private const string CONTENT_TYPE_GIF = "image/gif";
         private const string CONTENT_TYPE_WEBP = "image/webp";
-        private const long PREVIEW_IMAGE_SIZE_LIMIT = 500000;
+        private const long PREVIEW_IMAGE_SIZE_LIMIT = 750000;
```

Comment:
Is this needed?

Reply:
Image previews are gone, we load the full-resolution NFT now, so we needed to: 
- get rid of the limit
- up the limit

I believe we should keep a limit at least, wdyt?

Reply:
Ok, then the limit increase sounds good!